genie-agent/src/main/java/com/netflix/genie/agent/cli/JobRequestConverter.java

```diff
@@ -92,7 +92,7 @@ public AgentJobRequest agentJobRequestArgsToDTO(
             .Builder()
             .withRequestedJobDirectoryLocation(jobRequestArguments.getJobDirectoryLocation())
             .withTimeoutRequested(jobRequestArguments.getTimeout())
-            .withArchivingDisabled(jobRequestArguments.isArchivalDisabled())
+            .withArchiveLocation(jobRequestArguments.getArchiveLocation())
```

Comment:
This commit seems to be missing quite a few things.. maybe it was not meant for review yet?

Reply:
The persistence layer change that you were looking for @mprimi, is already there in the file  - genie-web/src/main/java/com/netflix/genie/web/jpa/services/JpaJobPersistenceServiceImpl.java

Reply:
There is a bug which I identified.. I am currently storing only the archiveLocation provided by the user from cli which is really the s3 prefix and not the actual archiveLocation with the jobFolder name appended. I will work on fixing that. Will request a review after that.